ifrs17/Import/Importers.ipynb

```diff
@@ -391,27 +417,23 @@
             "source": [
                 "public async Task DataNodeFactoryAsync(IDataSet dataSet, string tableName, ImportArgs args)",
                 "\n{",
-                "\n    var partition = (await DataSource.Query<PartitionByReportingNode>().Where(p => p.ReportingNode == args.ReportingNode && p.Scenario == null).ToArrayAsync()).SingleOrDefault();",
+                "\n    var partition = (await DataSource.Query<PartitionByReportingNode>().Where(p => p.ReportingNode == args.ReportingNode).ToArrayAsync()).SingleOrDefault();",
                 "\n    if(partition == null) { ApplicationMessage.Log(Error.ParsedPartitionNotFound); return; }",
```

Comment:
It wouldn't be possible to have more than one partition satisfying the where clause because there are listed all identity properties. Hence, I agree that we should only change SingleOrDefault -> FirstOrDefault

Reply:
FirstOrDefault implemented